streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -135,6 +140,10 @@ public void openDB(ProcessorContext context) {
         // (this could be a bug in the RocksDB code and their devs have been contacted).
         options.setIncreaseParallelism(Math.max(Runtime.getRuntime().availableProcessors(), 2));
 
+        if (prepareForBulkload) {
+            options.prepareForBulkLoad();
```

Comment:
There was some confusion over the number of times we  open and close the `RocksDBStateStore` for handling optimized bulk loads, once that was clarified the comments pertaining to setting `prepareForBulkload` and `open` didn't need to be addressed.